### PR TITLE
Fixes tank default release

### LINKED ISSUE #13733: Small o2 tanks don't start with 16 pressure.
I'm not entirely sure it's an issue, but recently someone changed so oxygen tanks start with 16 pressure release. Althought this change didn't affect small tanks, such as emergency oxygen tank, both extended and normal.

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Properly set to 16 now

Fixes #13733

### CODE REVIEW

File: code/game/objects/items/weapons/tanks/tanks.dm [hunk 1]
```diff
@@ -17,7 +17,7 @@
 	throw_range = 4
 
 	var/datum/gas_mixture/air_contents = null
-	var/distribute_pressure = ONE_ATMOSPHERE
+	var/distribute_pressure = 16
```
Comment (duncathan) [inline code/game/objects/items/weapons/tanks/tanks.dm]: Use the define

File: code/game/objects/items/weapons/tanks/tank_types.dm [hunk 2]
```diff
@@ -156,7 +156,7 @@
 	slot_flags = SLOT_BELT
 	w_class = 2
 	force = 4
-	distribute_pressure = ONE_ATMOSPHERE*O2STANDARD
+	distribute_pressure = 16
```
Comment (duncathan) [inline code/game/objects/items/weapons/tanks/tank_types.dm]: See above

**Comment by tkdrg** [comment]
I think defines would be better in the `__DEFINE` folder, but I'd like @duncathan 's opinion here as well.

File: code/game/objects/items/weapons/tanks/tanks.dm [hunk 3]
```diff
@@ -17,7 +13,7 @@
 	throw_range = 4
 
 	var/datum/gas_mixture/air_contents = null
-	var/distribute_pressure = ONE_ATMOSPHERE
+	var/distribute_pressure = TANK_DEFAULT_RELEASE_PRESSURE
```
Comment (duncathan) [inline code/game/objects/items/weapons/tanks/tanks.dm]: This one is supposed to be 101.325 kPa, not 16 kPa. Leave it as it was.

**Comment by duncathan** [comment]
Throw the defines in __DEFINES/atmospherics.dm

**Comment by lzimann** [comment]
:+1:

**Comment by KorPhaeron** [comment]
>merge conflicts on my one line change

Fuck it I'll just redo the pull

**Pull request closed without merging**